napalm_yang/mappings/ios/parsers/config/openconfig-if-ip/ipv4.yaml

```diff
@@ -1,34 +1,35 @@
 ---
 metadata:
-    processor: TextParser
+    processor: TextTree
 
 ipv4:
     _process: unnecessary
     config:
         _process: unnecessary
         enabled:
-            _process: # TODO look at this one, I think this is different depending on IOS version
+            _process:
                 - mode: is_absent
-                  regexp: "(?P<value>^\\s*switchport mode.*$)"
+                  path: "switchport"
         mtu:
             _process: not_implemented
     addresses:
         _process: unnecessary
         address:
             _process:
-                - regexp: "(?P<block>ip address (?P<key>(?P<ip>.*)) (?P<prefix>([0-255]|\\.)*)(?P<secondary> secondary)*)$"
+                - path: "ip.address.?prefix.?mask"
+                  regexp: "^(?P<value>\\d+\\.\\d+\\.\\d+\\.\\d+)"
```

Comment:
Ah, I think I got it. So with a question mark you basically assign whatever key is below `address` to a field name `prefix`.  But that key must match `regex`, plus `prefix` will be set to the `value` named capture group? And now you can reference that value by asking for `prefix`. And the same goes for `mask` as well. Correct?

But what will happen if I have `ip address 1.1.1.1 dog`?